### ISSUE TITLE
[macOS] mpv ignores its config folder, launches with defaults

### ISSUE THREAD

**Issue by for-coursera** [mps-youtube/mps-youtube#821]
## Issue

The problem is, `mpv` launched via `mpsyt` doesn't respect its configuration files in my home folder (ignores settings from `~/.config/mpv`).

After taking a look `mpsyt` source, I found this ([master branch, player.py:432](https://github.com/mps-youtube/mps-youtube/blob/3d94835bc97b1a4f4b994090b581fb22a83b9b56/mps_youtube/player.py#L432)):

    cmd.append('--input-conf=' + input_file)

Which somehow seems to be the culprit here – because if I comment it out, `mpv` then does find its configs and runs OK. However, commenting it out also breaks many other things, as I can see (like hotkeys), so, it's there for some really important purpose, I presume :).

Now, I've tried to `set playerargs --config-dir=/Users/me/.config/mpv/` in the `mpsyt`. I've also tried to add that setting directly to `cmd` at `player.py` just below `--input-conf`:
    cmd.append('--input-conf=' + input_file)
    cmd.append('--config-dir=/Users/me/.config/mpv/`)

But sadly neither of my attempts succeeded. (I've tried the development version, too, but it fails at playing (`AttributeError: 'mpv' object has no attribute 'mpv_usesock'`))

Thus, my question: is there any way to fix that? *Somehow* related to the problem may be the fact that I can't really upgrade `mpv` – because my macOS is too old, and all the `mpv` latest [builds](https://laboratory.stolendata.net/~djinn/mpv_osx/) just fail at starting up.

Thanks :).

## Your Environment

    mpsyt --version
    mpsyt version      : 0.2.8
       notes           : released 17 February 2018
    pafy version       : 0.5.4 (youtube-dl backend)
    youtube-dl version : 2018.03.20
    Python version     : 3.6.3 (default, Dec  7 2017, 11:35:35)
    [GCC 4.2.1 Compatible Apple LLVM 5.1 (clang-503.0.40)]
    Processor          : i386
    Machine type       : x86_64
    Architecture       : 64bit,
    Platform           : Darwin-12.6.0-x86_64-i386-64bit
    sys.stdout.enc     : UTF-8
    default enc        : utf-8
    Config dir         : /Users/me/.config/mps-youtube
    env:TERM           : xterm-256color
    env:SHELL          : /bin/bash
    env:LANG           : en_US.UTF-8
    
Now, to the outdated software :)
    
    mpv --version
    mpv git-d1a46c2 (C) 2000-2015 mpv/MPlayer/mplayer2 projects
     built on Sun Oct 25 01:03:36 CEST 2015
    ffmpeg library versions:
       libavutil       55.4.100
       libavcodec      57.9.100
       libavformat     57.10.101
       libswscale      4.0.100
       libavfilter     6.13.100
       libswresample   2.0.100
    ffmpeg version: N-76238-ge06ef9a

    macOS 10.8.5

**Comment by indeedwatson** [comment]
Is this going to get addressed at all? It's a rather odd behavior. I'm having the same issue on arch linux.

**Comment by for-coursera** [comment]
@indeedwatson, What exactly have you found?

**Comment by indeedwatson** [comment]
@for-coursera Not sure if your issue is the same but I hope it helps https://github.com/mps-youtube/mps-youtube/wiki/Troubleshooting#custom-mplayer-or-mpv-keyboard-shortcuts-are-ignored

**Comment by for-coursera** [comment]
@indeedwatson, Yeah, thanks. Seems like that should be the fix :) (I'm actually not using mpsyt any more, so can't really verify). Still, I agree with you, it should use the user's input.conf by default (or at lest check for it).
